src/main/java/com/axonivy/market/constants/GitHubConstants.java

```diff
@@ -8,5 +8,6 @@ public class GitHubConstants {
 	public static final String AXONIVY_MARKET_ORGANIZATION_NAME = "axonivy-market";
 	public static final String AXONIVY_MARKETPLACE_REPO_NAME = "market";
 	public static final String AXONIVY_MARKETPLACE_PATH = "market";
+  public static final String DEFAULT_BRANCH = "feature/MARP-463-Multilingualism-for-Website";
```

Comment:
update default branch

Reply:
Ok, I will update it later when new meta JSON files are applied on the market repo.